packages/presentation/lib/screen/home/home_page.dart

```diff
@@ -122,16 +100,19 @@ class _HomePageState extends BlocScreenState<HomePage, HomeBloc> {
                               ? null
                               : () {
                                   bloc.submit();
+                                  bloc.makeAttemptsTextAvailable(
+                                      blocData.gameState);
                                 },
                           child: const Text('Submit'),
                         ),
                       ],
                     ),
-                    if (blocData.attempts >= 1)
+                    if (blocData.isAttemptsTextAvailable == true) ...[
                       Text(
                         'Try again. You have ${3 - blocData.attempts} more attempts',
```

Comment:
create field attemptsLeft in state. this property will have a getter which will return the value `3 - attempts`. in this case we will remove some logic from ui

Reply:
resolved